pytorch_lightning/core/lightning.py

```diff
@@ -1927,8 +1927,6 @@ def add_to_queue(self, queue: pl.plugins.training_type.ddp_spawn._FakeQueue) ->
             This method was deprecated in v1.5 in favor of `DDPSpawnPlugin.add_to_queue`
             and will be removed in v1.7.
         """
-        if self.trainer and isinstance(self.trainer.training_type_plugin, pl.plugins.training_type.DDPSpawnPlugin):
```

Comment:
I'm a bit confused. so now there will be no default impl. of add_to_queue/get_from_queue in LM, we will only rely on user overrides?

Reply:
there is, the default implementation is in  the plugin still, it just doesn't get reroute through the LM anymore.